pipeline/post-results.groovy

```diff
@@ -80,9 +82,25 @@ node(nodeName) {
             }
 
             def testStatus = "ABORTED"
+            /* Publish results through E-mail and Google Chat */
+            def tag = msgMap["pipeline"]["tags"]
```

Comment:
Can you add a check whether tags key is present before assigning tierLevel and stageLevel as well before this line?

Reply:
added the check for tags